Jenkinsfile

```diff
@@ -66,4 +64,67 @@ node {
       }
     }
   }
+
+  stage ("Run Akamai Production Smoke Tests") {
+    try {
+      if (RUN_SMOKE_TESTS) {
+        openShiftUtils.withNode {
+          withCredentials([
+            string(credentialsId: "vaultRoleId", variable: 'DYNACONF_IQE_VAULT_ROLE_ID'),
+            string(credentialsId: "vaultSecretId", variable: 'DYNACONF_IQE_VAULT_SECRET_ID'),
+          ]) {
+            // set some env variables for authentication in the iqe-core pod
+            withEnv([
+              "DYNACONF_IQE_VAULT_ROLE_ID=$DYNACONF_IQE_VAULT_ROLE_ID",
+              "DYNACONF_IQE_VAULT_SECRET_ID=$DYNACONF_IQE_VAULT_SECRET_ID",
+              "DYNACONF_IQE_VAULT_LOADER_ENABLED=true",
+              "ENV_FOR_DYNACONF=prod"
+            ]) {
+              // install akamai and 3scale plugins, run smoke tests
+              sh "iqe plugin install akamai 3scale"
+              sh "IQE_AKAMAI_CERTIFI=true DYNACONF_AKAMAI=\'@json {\"release\":\"${RELEASESTR}\"}\' iqe tests plugin akamai -s -m ${PRODTESTSTR}"
+              sh "iqe tests plugin akamai -k 'test_api.py' -m prod"
+              sh "iqe tests plugin 3scale --akamai-production -m akamai_smoke"
+            }
+          }
+        }
+      }
+      else {
+        sh "echo Smoke tests cannot run against STAGE environment as it requires VPN connection"
+      }
+    } catch(e) {
+      // If the tests don't all pass, roll back changes:
+      // Re-upload the old main.yml/json files
+      configFileProvider([configFile(fileId: "9f0c91bc-4feb-4076-9f3e-13da94ff3cef", variable: "AKAMAI_HOST_KEY")]) {
+        sh "rm main.yml"
+        sh "cp main.yml.bak main.yml"
+        sh "rm releases.yml"
+        sh "cp releases.yml.bak releases.yml"
+        sh "rm -r chrome"
+        sh "cp -r chrome.bak chrome"
+        withCredentials(bindings: [sshUserPrivateKey(credentialsId: "cloud-netstorage",
+                keyFileVariable: "privateKeyFile",
+                passphraseVariable: "",
+                usernameVariable: "")]) {
+          sh """
+            eval `ssh-agent`
+            ssh-add \"$privateKeyFile\"
+            cp \"$AKAMAI_HOST_KEY\" ~/.ssh/known_hosts
+            chmod 600 ~/.ssh/known_hosts
+            rsync -arv -e \"ssh -2\" *.yml sshacs@cloud-unprotected.upload.akamai.com:${AKAMAI_APP_PATH}
+            rsync -arv -e \"ssh -2\" ./chrome/*.json sshacs@cloud-unprotected.upload.akamai.com:${AKAMAI_APP_PATH}/chrome
+          """
+        }
+      }
+
+      // If smoke tests have errors
+      currentBuild.result = 'ABORTED'
+      mail body: "CSC smoke tests failed. Failed build is here: ${env.BUILD_URL}" ,
+            from: 'csc-jenkins@redhat.com',
+            replyTo: 'rlong@redhat.com',
+            subject: 'CSC build: Smoke tests failed',
+            to: 'rlong@redhat.com,khala@redhat.com,jdupuy@redhat.com'
```

Comment:
@john-dupuy what are your thoughts on creating a mailing list something like "CRC outages" and funneling all of these errors to this place?

Reply:
I've opened a ticket with IT to create the mailing list and will change this message to use the list in a follow-up.

Reply:
Actually, seems the list has already been created, I'll make this change now.